java.lang.NoClassDefFoundError: com/intellij/refactoring/JavaRefactoringSettings

### Type of JetBrains IDE (IntelliJ, PyCharm, etc.) and version
PyCharm 2023.2 (build PY-232.8660.197)

### Operating System 
Linux 6.4.6-76060406-generic (amd64)

### TeXiFy IDEA version
0.9.1

### Description
I tried to inline a command and it freaked out. The command has two mandatory arguments


### Stacktrace
```
java.lang.NoClassDefFoundError: com/intellij/refactoring/JavaRefactoringSettings
	at nl.hannahsten.texifyidea.refactoring.inlinecommand.LatexInlineDialog.isInlineThis(LatexInlineDialog.kt:34)
	at com.intellij.refactoring.inline.InlineOptionsDialog.createCenterPanel(InlineOptionsDialog.java:107)
	at com.intellij.openapi.ui.DialogWrapper.init(DialogWrapper.java:1312)
	at nl.hannahsten.texifyidea.refactoring.inlinecommand.LatexInlineCommandDialog.<init>(LatexInlineCommandDialog.kt:26)
	at nl.hannahsten.texifyidea.refactoring.inlinecommand.LatexInlineCommandHandler.inlineElement(LatexInlineCommandHandler.kt:32)
	at com.intellij.refactoring.inline.InlineRefactoringActionHandler.invoke(InlineRefactoringActionHandler.java:58)
	at com.intellij.refactoring.actions.BaseRefactoringAction.performRefactoringAction(BaseRefactoringAction.java:161)
	at com.intellij.refactoring.actions.BaseRefactoringAction.actionPerformed(BaseRefactoringAction.java:110)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:339)
	at com.intellij.openapi.keymap.impl.ActionProcessor.performAction(ActionProcessor.java:47)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$myActionProcessor$1.performAction(IdeKeyEventDispatcher.kt:502)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcherKt.doPerformActionInner$lambda$5$lambda$4(IdeKeyEventDispatcher.kt:865)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcherKt.doPerformActionInner$lambda$5(IdeKeyEventDispatcher.kt:865)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:362)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcherKt.doPerformActionInner(IdeKeyEventDispatcher.kt:863)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcherKt.access$doPerformActionInner(IdeKeyEventDispatcher.kt:1)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction$intellij_platform_ide_impl(IdeKeyEventDispatcher.kt:587)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.kt:513)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.kt:454)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.kt:447)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.kt:309)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.kt:617)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:587)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:992)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:992)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: com.intellij.refactoring.JavaRefactoringSettings PluginClassLoader(plugin=PluginDescriptor(name=TeXiFy IDEA, id=nl.rubensten.texifyidea, descriptorPath=plugin.xml, path=~/.local/share/JetBrains/PyCharm2023.2/TeXiFy-IDEA, version=0.9.1, package=null, isBundled=false), packagePrefix=null, state=active)
	... 43 more

```

--- a/src/nl/hannahsten/texifyidea/refactoring/inlinecommand/LatexInlineCommandDescriptor.kt
+++ b/src/nl/hannahsten/texifyidea/refactoring/inlinecommand/LatexInlineCommandDescriptor.kt
@@ -1,7 +1,7 @@
 package nl.hannahsten.texifyidea.refactoring.inlinecommand
 
-import com.intellij.java.refactoring.JavaRefactoringBundle
 import com.intellij.psi.PsiElement
+import com.intellij.refactoring.RefactoringBundle
 import com.intellij.usageView.UsageViewBundle
 import com.intellij.usageView.UsageViewDescriptor
 import nl.hannahsten.texifyidea.file.LatexFile
@@ -27,7 +27,7 @@ class LatexInlineCommandDescriptor(private val myElement: PsiElement) : UsageVie
     }
 
     override fun getCodeReferencesText(usagesCount: Int, filesCount: Int): String {
-        return JavaRefactoringBundle.message(
+        return RefactoringBundle.message(
             "invocations.to.be.inlined",
             UsageViewBundle.getReferencesString(usagesCount, filesCount)
         )

--- a/src/nl/hannahsten/texifyidea/refactoring/inlinecommand/LatexInlineCommandDialog.kt
+++ b/src/nl/hannahsten/texifyidea/refactoring/inlinecommand/LatexInlineCommandDialog.kt
@@ -54,7 +54,6 @@ class LatexInlineCommandDialog(
                 GlobalSearchScope.projectScope(myProject)
             )
         )
-        updateSettingsPreferences()
     }
 
     override fun getNumberOfOccurrences(): Int {

--- a/src/nl/hannahsten/texifyidea/refactoring/inlinecommand/LatexInlineDialog.kt
+++ b/src/nl/hannahsten/texifyidea/refactoring/inlinecommand/LatexInlineDialog.kt
@@ -4,7 +4,6 @@ import com.intellij.openapi.project.Project
 import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiNameIdentifierOwner
 import com.intellij.psi.search.searches.ReferencesSearch
-import com.intellij.refactoring.JavaRefactoringSettings
 import com.intellij.refactoring.RefactoringBundle
 import com.intellij.refactoring.inline.InlineOptionsDialog
 
@@ -30,13 +29,7 @@ abstract class LatexInlineDialog(project: Project?, genericDefinition: PsiElemen
         return RefactoringBundle.message("inline.method.border.title")
     }
 
-    override fun isInlineThis(): Boolean {
-        return JavaRefactoringSettings.getInstance().INLINE_METHOD_THIS
-    }
-
-    override fun isKeepTheDeclarationByDefault(): Boolean {
-        return JavaRefactoringSettings.getInstance().INLINE_METHOD_KEEP
-    }
+    override fun isInlineThis(): Boolean = false
 
     override fun hasHelpAction(): Boolean {
         return false
@@ -44,15 +37,5 @@ abstract class LatexInlineDialog(project: Project?, genericDefinition: PsiElemen
 
     override fun allowInlineAll(): Boolean {
         return true
-    }
-
-    protected fun updateSettingsPreferences() {
-        val settings = JavaRefactoringSettings.getInstance()
-        if (myRbInlineThisOnly.isEnabled && myRbInlineAll.isEnabled) {
-            settings.INLINE_METHOD_THIS = isInlineThisOnly
-        }
-        if (myKeepTheDeclaration != null && myKeepTheDeclaration!!.isEnabled) {
-            settings.INLINE_METHOD_KEEP = isKeepTheDeclaration
-        }
     }
 }

--- a/src/nl/hannahsten/texifyidea/refactoring/inlinefile/LatexInlineFileDescriptor.kt
+++ b/src/nl/hannahsten/texifyidea/refactoring/inlinefile/LatexInlineFileDescriptor.kt
@@ -1,7 +1,7 @@
 package nl.hannahsten.texifyidea.refactoring.inlinefile
 
-import com.intellij.java.refactoring.JavaRefactoringBundle
 import com.intellij.psi.PsiElement
+import com.intellij.refactoring.RefactoringBundle
 import com.intellij.usageView.UsageViewBundle
 import com.intellij.usageView.UsageViewDescriptor
 import nl.hannahsten.texifyidea.file.LatexFile
@@ -27,7 +27,7 @@ class LatexInlineFileDescriptor(private val myElement: PsiElement) : UsageViewDe
     }
 
     override fun getCodeReferencesText(usagesCount: Int, filesCount: Int): String {
-        return JavaRefactoringBundle.message(
+        return RefactoringBundle.message(
             "invocations.to.be.inlined",
             UsageViewBundle.getReferencesString(usagesCount, filesCount)
         )

--- a/src/nl/hannahsten/texifyidea/refactoring/inlinefile/LatexInlineFileDialog.kt
+++ b/src/nl/hannahsten/texifyidea/refactoring/inlinefile/LatexInlineFileDialog.kt
@@ -55,7 +55,6 @@ class LatexInlineFileDialog(
                 GlobalSearchScope.projectScope(myProject)
             )
         )
-        updateSettingsPreferences()
     }
 
     override fun getNumberOfOccurrences(): Int {

